Objects/sliceobject.c

```diff
@@ -103,12 +103,20 @@ PyObject _Py_EllipsisObject = _PyObject_HEAD_INIT(&PyEllipsis_Type);
 
 /* Slice object implementation */
 
+void _PySlice_ClearCache(_PyFreeListState *state)
+{
+    PySliceObject *obj = state->slice_state.slice_cache;
+    if (obj != NULL) {
+        state->slice_state.slice_cache = NULL;
+        PyObject_GC_Del(obj);
+    }
+}
 
-void _PySlice_Fini(PyInterpreterState *interp)
+void _PySlice_Fini(_PyFreeListState *state)
```

Comment:
This is identical to `_PySlice_ClearCache`. Do we need both functions? If you want to keep both names, maybe have one call the other.

Reply:
Let's keep both names; I don't want to confuse people who work on the default build.